EIPS/eip-7002.md

```diff
@@ -706,6 +706,10 @@ Note: the above code reverts if the fee is too high, the fee can change signific
 
 Using an EOA to request withdrawals will always result in overpayment of fees. There is no way for an EOA to use a wrapper contract to request a withdrawal. And even if a way existed, the gas cost of returning the overage would likely be higher than the overage itself. If requesting withdrawals to an EOA through the system contract is desired, we recommend that users perform transaction simulations to estimate a reasonable fee amount to send.
 
+### System Call failure
+
+Although the likelihood of a failed system call to SYSTEM_ADDRESS is extremely low, the behavior in such cases is well-defined: the block is marked as invalid. However, if the failure results from processing a transaction within the block, the public mempool may still retain the transaction even after the block is invalidated. This can result in the offending transaction being included again, potentially causing one or more subsequent slots to go without valid blocks. To mitigate this, we recommend that the block producer implementation shuffle their transaction set to increase the chances of producing a valid block, without the offending transaction(s). The block producer implementation and/or the mempool should be aware of system call failure scenarios to enable this behavior.
```

Comment:
Same here, `SYSTEM_ADDRESS` does not (should not) have code, but `WITHDRAWAL_REQUEST_PREDEPLOY_ADDRESS` has the relevant code here.